File: 01.JAVA/pg-166-169/pg_166_07.java
import java.util.Scanner;

public class pg_166_07
{
    public static void main(String[] args) {
        {
            Scanner enter = new Scanner(System.in);
            int num;
            int i = 2;
            boolean prime = true;
            
            System.out.println("");
            System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
            System.out.println("READ A TWO-DIGITS INTEGER AND DETERMINE IF IT'S A PRIME AND NEGATIVE NUMBER");
            System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
            System.out.println("");

            System.out.print("Write a two-digits iteger: ");
            num = enter.nextInt();

            if(num < 0)
            {
                System.out.print(" The written number is negative");
                num *= -1;
            }
            else
                {
                    System.out.print("The written number is positive");
                }

            if(num >= 10 && num <= 99)
            {
                while( i < num)
                {
                    if(num % i == 0)
                    {
                        prime = false;
                        break;
                    }
                    i++;
                }

                if(prime)
                {
                    System.out.println(" and is prime. ");
                }
                else
                {
                    System.out.println(" and isn't prime. ");
                }
            }
            else
            {
                System.out.println(" but doesn't have two digits. Please try again!");
            }
        }
    }
}

File: 02.PYTHON/pg-166-169/pg_166_07.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("READ A TWO-DIGITS INTEGER AND DETERMINE IF IT'S A PRIME AND NEGATIVE NUMBER");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("");

num = int(input("Write a two-digits integer: "))
i = 2;
prime = True;

if(num < 0):
    
        print("The written number is negative ");
        num *= -1;

else:
        print("The written number is positive ");

if(num >= 10 and num <= 99):
        
        while(i < num):
                if(num % i == 0):
                        prime = False;
                        break;
                i += 1;
        if(prime):
                print("and is prime.")
        else:
                print("and isn't prime.")
else:
        print("but doesn't have two digits. Please try again!")